torchft/local_sgd.py

```diff
@@ -182,6 +183,8 @@ class DiLoCo(LocalSGD):
     diloco: https://arxiv.org/pdf/2311.08105
     """
 
+    BUCKET_SIZE_BYTES = 32 * 1024 * 1024
```

Comment:
Can we make this configurable parameter named `bucket_cap_mb` to match DDP? 

https://pytorch.org/docs/stable/generated/torch.nn.parallel.DistributedDataParallel.html